packages/charts/examples/components/demo/advanced-usage/events/event-sampler/event-sampler.component.html

```diff
@@ -13,14 +13,13 @@
                                [seriesRenderState]="chartAssist.renderStatesIndex[legendSeries.id]?.state"
```

Comment:
This is as high in the file as github will allow me to comment, but it would be good to use the titlecase pipe for the descriptionPrimary above.

Also, the `seriesUnitLabel` attribute on line 6 can be removed since we're no longer showing a value.